htdocs/app/view/login/Readme.md

```diff
@@ -0,0 +1 @@
+This folder contains the Login view and controller.
```

Comment:
This readme is unnecessary - usually it is only needed as a placeholder for an empty directory. Git doesn't track directories by themselves, only files.